documentation/book/security.adoc

```diff
@@ -1,8 +1,8 @@
 == Security
 
-The Strimzi project supports data encryption of communication between the different Kafka and Strimzi components (Kafka brokers, Zookeeper nodes, Kafka Connect and Strimzi Topic Operator) by means of the SSL/TLS protocol.
+The {ProductName} project supports data encryption of communication between the different Kafka and {ProductName} components (Kafka brokers, Zookeeper nodes, Kafka Connect and {ProductName} Topic Operator) by means of the SSL/TLS protocol.
```

Comment:
Should we say more generally `Entity Operator` here? Now we have the `User Operator` as well which uses the TLS sidecar to communicate with Zookeeper with encryption. Wdyt @scholzj @tombentley ?

Reply:
Or maybe just saying "and so on" or specifying "Topic Operator and User Operator"

Reply:
I would just remove the list of compoenents and leave only _Kafka and {ProductName} components_